templates/release-docs-changelog.yml

```diff
@@ -0,0 +1,78 @@
+spec:
+  inputs:
+    stage:
+      description: The pipeline stage where to perform the docs release
+      default: publish
+    runner:
+      description: GitLab CI runner tag
+      default: hetzner-amd-beefy
+    alpine-git-tag:
+      description: Tag for alpine/git container image
+      default: latest
+    remote_changelog_file:
+      description: Remote changelog file to be used
+---
+
+
+release:mender-docs-changelog:
+  stage: $[[ inputs.stage ]]
+  tags:
+    - $[[ inputs.runner ]]
+  needs: []
+  image:
+    name: alpine/git:$[[ inputs.alpine-git-tag ]]
+    entrypoint: [""]
+  rules:
+    # Only make available for stable tags
+    - if: '$CI_COMMIT_TAG =~ /^v?\d+\.\d+\.\d+$/'
+      allow_failure: true
+  variables:
+    GITHUB_CHANGELOG_REPO_URL: "mender-docs-changelog"
+  before_script:
+    # Check required input parameters
+    - |
+      if [ -z "$[[ inputs.remote_changelog_file ]]" ]; then
+        echo "ERROR: Missing template argument 'remote_changelog_file'"
+        echo "Please specify the remote changelog name in your include section:"
+        echo "  inputs:"
+        echo "    remote_changelog_file: PATH_TO_CHANGELOG_FILE"
+        exit 1
+      fi
```

Comment:
This block is not necessary. This is a new learning from my tests. See  https://github.com/mendersoftware/mendertesting/pull/389/commits/4ff0d82863d047e993244dcf4e395f803da71c6b